packages/ui/src/components/TagInput/index.tsx

```diff
@@ -1,13 +1,33 @@
-import type { Theme } from '@emotion/react'
 import styled from '@emotion/styled'
+import { Icon } from '@ultraviolet/icons'
 import type {
   ChangeEvent,
   ClipboardEventHandler,
   KeyboardEventHandler,
+  ReactNode,
 } from 'react'
-import { useEffect, useRef, useState } from 'react'
+import { useEffect, useId, useMemo, useRef, useState } from 'react'
 import { getUUID } from '../../utils'
+import { Button } from '../Button'
+import { Stack } from '../Stack'
 import { Tag } from '../Tag'
+import { Text } from '../Text'
+import { Tooltip } from '../Tooltip'
+
+// SIZE
+export const TAGINPUT_SIZE_HEIGHT = {
+  large: 48,
+  medium: 40,
+  small: 32,
```

Comment:
There is a little issue with the sizes, I think it should subtract the border so it will match other inputs

TagInput:
<img width="973" alt="Screenshot 2024-02-07 at 18 06 42" src="https://github.com/scaleway/ultraviolet/assets/15812968/406bb581-6f0f-44d9-94f0-2cc0542b0aec">

TextInput:
<img width="980" alt="Screenshot 2024-02-07 at 18 06 57" src="https://github.com/scaleway/ultraviolet/assets/15812968/8efb6072-8d38-4bce-a32f-01bd8c633ff5">

So large should be 46, medium 38 and small 30